site/docs/reference/Connectors/capture-connectors/google-cloud-sql-sqlserver.md

```diff
@@ -0,0 +1,149 @@
+---
+sidebar_position: 6
+---
+# Google Cloud SQL for SQL Server
+
+This connector uses change data capture (CDC) to continuously capture updates in a Microsoft SQL Server database into one or more Flow collections.
+
+It’s available for use in the Flow web application. For local development or open-source workflows, [`ghcr.io/estuary/source-sqlserver:dev`](https://ghcr.io/estuary/source-sqlserver:dev) provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+
+## Supported versions and platforms
+
+This connector supports SQL Server 2017 and later on major cloud providers,
+as well as self-hosted instances.
+Setup instructions are provided for the following platforms:
+
+* [Self-hosted SQL Server](#setup-self-hosted-sql-server)
+* [Azure SQL Database](#setup-azure-sql-database)
+* [Amazon RDS for SQL Server](#setup-amazon-rds-for-sql-server)
+* [Google Cloud SQL for SQL Server](#setup-google-cloud-sql-for-sql-server)
+
+## Prerequisites
+
+To capture change events from SQL Server tables using this connector, you need:
+
+* For each table to be captured, a primary key should be specified in the database.
+If a table doesn't have a primary key, you must manually specify a key in the associated Flow collection definition while creating the capture.
+[See detailed steps](#specifying-flow-collection-keys).
+
+* [CDC enabled](https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/enable-and-disable-change-data-capture-sql-server?view=sql-server-ver16)
+on the database and the individual tables to be captured.
+(This creates *change tables* in the database, from which the connector reads.)
+
+* A **watermarks table**. The watermarks table is a small “scratch space” to which the connector occasionally writes a small amount of data to ensure accuracy when backfilling preexisting table contents.
+
+* A user role with:
+  * `SELECT` permissions on the CDC schema and the schemas that contain tables to be captured.
+  * Access to the change tables created as part of the SQL Server CDC process.
+  * `SELECT`, `INSERT`, and `UPDATE` permissions on the watermarks table
+
+### Setup: Google Cloud SQL for SQL Server
```

Comment:
We could probably just rename this "Setup" cause it's the only setup in this docs page now

Reply:
addressed